rdmo/core/settings.py

```diff
@@ -99,7 +99,14 @@
 ACCOUNT = False
 ACCOUNT_SIGNUP = False
 ACCOUNT_GROUPS = []
+
 ACCOUNT_TERMS_OF_USE = False
+ACCOUNT_TERMS_OF_USE_MIDDLEWARE = 'rdmo.accounts.middleware.TermsAndConditionsRedirectMiddleware'
```

Comment:
I would skip the `ACCOUNT_TERMS_OF_USE_MIDDLEWARE` setting, the middleware needs to be added anyway, we could add it by default. (Did I miss where you did this already?)

Reply:
You don't think that there will be a use-case where somebody wants to implement their own middleware for this? Ok, I guess not..

I didn't add it by default to the middlewares yet but seems plausible since the middleware only has an effect with `ACCOUNT_TERMS_OF_USE` enabled, I'll add it then.

Reply:
Yes, the one `if` is fine for everybody, and when you want to add your own middleware, you can do so by changing `MIDDLEWARE` what people already need to do for allauth and Shibboleth.